web/src/main/java/org/springframework/security/web/authentication/DelegatingAuthenticationConverter.java

```diff
@@ -38,12 +37,11 @@ public final class DelegatingAuthenticationConverter implements AuthenticationCo
 
 	public DelegatingAuthenticationConverter(List<AuthenticationConverter> delegates) {
 		Assert.notEmpty(delegates, "delegates cannot be null");
-		this.delegates = new ArrayList<>(delegates);
+		this.delegates = List.copyOf(delegates);
 	}
 
 	public DelegatingAuthenticationConverter(AuthenticationConverter... delegates) {
-		Assert.notEmpty(delegates, "delegates cannot be null");
-		this.delegates = List.of(delegates);
+		this(List.of(delegates));
```

Comment:
I'm not a huge fan of recreating the list twice. Could we please leave this constructor as-is, even though that duplicates the null check?